src/stylus/react-selectize.styl

```diff
@@ -0,0 +1,929 @@
+.example .react-selectize.multi-select .removeable-emoji {
```

Comment:
Apparently, I should be able to add this line to add all styling for `react-selectize`:

@import 'node_modules/react-selectize/themes/index.css'

However, this wasn't working for me, so I copied and pasted all the `css` from the `react-selectize` examples page...